packages/cli/command_spec.pony

```diff
@@ -325,22 +343,66 @@ class val ArgSpec
     _name + "[" + _typ.string() + "]" +
       if not _required then "(=" + _default.string() + ")" else "" end
 
-type _Value is (Bool | String | I64 | F64)
+class _StringSeq is ReadSeq[String]
```

Comment:
This looks like a persistent data structure holding a sequence of strings. Have you thought about using `Vec[String]` from the `collections/persistent` package instead of rolling your own here?